pkgs/applications/misc/plasma-theme-switcher/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  stdenv, lib, cmake, extra-cmake-modules, fetchFromGitHub, qtbase, kdeFrameworks
+}:
+
+stdenv.mkDerivation rec {
+  pname = "plasma-theme-switcher";
+  version = "0.1";
+  dontWrapQtApps = true;
+
+  src = fetchFromGitHub {
+    owner = "maldoinc";
+    repo = "plasma-theme-switcher";
+    rev = "v${version}";
+    sha256 = "sdcJ6K5QmglJEDIEl4sd8x7DuCPCqMHRxdYbcToM46Q=";
+  };
+
+  buildInputs = [ qtbase kdeFrameworks.plasma-framework ];
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=" ];
```

Comment:
`DCMAKE_BUILD_TYPE` is implicitly set by cmake already. Also, I dont think the `DCMAKE_INSTALL_PREFIX` option does anything, since `installPhase` is overwritten.
```suggestion
```

Reply:
Will test this tomorrow, to be honest I just looked at what other packages were doing - first time creating a package :p